packages/react-core/src/components/Dropdown/Toggle.tsx

```diff
@@ -109,7 +109,7 @@ export class Toggle extends React.Component<ToggleProps> {
     event.preventDefault();
     if ((event.key === 'Tab' || event.key === 'Enter' || event.key === ' ') && this.props.isOpen) {
       this.props.onToggle(!this.props.isOpen, event);
-    } else if ((event.key === 'Enter' || event.key === ' ') && !this.props.isOpen) {
+    } else if ((event.key === 'Enter' || event.key === ' ' || event.keyCode === 40) && !this.props.isOpen) {
```

Comment:
Change to `event.key === 'ArrowDown`

Reply:
I tried this first, but for some reason it was triggering Shift to open the dropdown as well. Using the keyCode made it so that only the down arrow key worked for it.